### PR TITLE
Created a user signup form #13

### PR DESCRIPTION
**Pull request by abdullah956**
#13 

-> created a branch user-signup-form.
-> made a forms.py in customer and added a signup form.
-> in urls.py added path to signup.
-> in veiws.py wrote a signup function.
-> in templates/customer added signup.html to make a user sign up form.

### CODE REVIEW

**Review by AbdulRahmanMudasser — COMMENTED** [review]
I have done some changes

**Comment by abdullah956** [comment]
@muzzammilh all the errorrs have been resolved. You can review the PR.

File: customer/urls.py [hunk 1]
```diff
@@ -3,5 +3,10 @@
 from . import views
 
 urlpatterns = [
-    path("test/", views.index, name="index"),
+    path("test/", views.home, name="home"),
```
Comment (muzzammilh) [inline customer/urls.py:6]: Change the URL to Home

**Review by muzzammilh — APPROVED** [review]
Just a minor change in url required.

**Comment by abdullah956** [comment]
> Just a minor change in url required.

The change in urls.py is done. Approval required.

**Review by muzzammilh — APPROVED** [review]

**Comment by muzzammilh** [comment]
@abdullah956 @AbdulRahmanMudasser Approved, please merge.

**Pull request merged**